server/utils/nunjucksSetup.ts

```diff
@@ -94,11 +94,13 @@ export function registerNunjucks(app?: express.Express): Environment {
     }
 
     const duration = intervalToDuration({ start: dob, end: today })
+
     let age = ''
-    if (duration.years < 1) {
-      age = formatDuration(duration, { format: ['months'], zero: true })
-    } else {
+    if (duration.years) {
       age = formatDuration(duration, { format: ['years'] })
+    } else {
+      // workaround below for Duration zero/undefined change (https://github.com/date-fns/date-fns/issues/3658)
```

Comment:
I've looked into this `date-fns` breaking change a bit more, @psoleckimoj, and re-ordered the code to allow a simpler workaround.

(I've linked to an issue someone else has raised about this. Does seem an odd change, to not distinguish between `undefined` and `0` in the return type 🤷‍♂️)